examples/magnum/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Should be `3.2`:
- https://github.com/ruslo/hunter/blob/v0.23.99/examples/foo/CMakeLists.txt#L4